examples/reference/publication_quality/bokeh/bar.py

```diff
@@ -0,0 +1,209 @@
+# ============================================================================
+# Publication-Quality Bar Chart - HoloViews Best Practices Example
+# ============================================================================
+# Demonstrates:
+# - Data extraction, transformation, and visualization separation
+# - Custom Bokeh themes for consistent styling
+# - Interactive tooltips with formatted data
+# - Text annotations on bars
+# - Professional fonts, grids, and axis formatting
+# - Panel integration for web serving
+# ============================================================================
+
+import panel as pn
+import hvsampledata
+import holoviews as hv
+
+from bokeh.models.formatters import NumeralTickFormatter
+from bokeh.themes import Theme
+from holoviews.plotting.bokeh import ElementPlot
+
+# ============================================================================
+# BOKEH THEME SETUP - Define global styling
+# ============================================================================
+
+ACCENT_COLOR = '#007ACC'  # Professional blue
+FONT_FAMILY = 'Roboto'
+
+def create_bokeh_theme(font_family=FONT_FAMILY, accent_color=ACCENT_COLOR):
+    """Create custom theme with specified font. Default: Roboto"""
+    return Theme(json={
+        'attrs': {
+            'Title': {
+                'text_font': font_family,
+                'text_font_size': '16pt',
+                'text_font_style': 'bold'
+            },
+            'Axis': {
+                'axis_label_text_font': font_family,
+                'axis_label_text_font_size': '12pt',
+                'axis_label_text_font_style': 'bold',
```

Comment:
I don't think the default `italic` style is readable or modern. `bold` is much better in my opinion.